Add HTTP example

hanslovsky opened the pull request.

hanslovsky committed:
Add HTTP example

hanslovsky committed:
Remove unused classes

hanslovsky committed:
Rename and restructure IntervalKeyLoaderAsLongKeyLoader

hanslovsky committed:
Simplify HTTPLoader

tpietzsch commented:
@hanslovsky The `IntervalKeyLoaderAsLongKeyLoader` is now part of imglib2-cache 1.0.0-beta-4.
Could you update this PR accordingly?

hanslovsky committed:
Update imglib2-cache dependency version to latest release

 - Use IntervalKeyLoaderAsLongKeyLoader from imglib2-cache
 - Remove obsolete comments

hanslovsky commented:
@tpietzsch Updated the PR in  c49b261

tpietzsch commented:
thank you @hanslovsky !
